ci/slt/pipeline.template.yml

```diff
@@ -29,6 +29,19 @@ steps:
           composition: sqllogictest
           run: sqllogictest
 
+  - id: sqllogictest-1-replica
```

Comment:
I think this belongs in Nightly, with the "testdrive 4 replicas" jobs and the like.

Reply:
I spaced out again, I thought you are adding it to the default CI pipeline :-( Please ignore me for the rest of the day.